ci/ci/ci.py

```diff
@@ -41,51 +42,51 @@
     for (index, [bss, deployable]) in enumerate(json.loads(os.environ.get('HAIL_WATCHED_BRANCHES', '[]')))
 ]
 
+dev_to_gh_user = {'dgoldste': 'daniel-goldstein'}
```

Comment:
Add information to AUTHORIZED_USERS or drive AUTHORIZED_USERS from some new structure that maps between these.